### ISSUE TITLE
Missing -webkit- prefix for text-decoration shorthand property 

### ISSUE THREAD

**Issue by simevidas** [postcss/autoprefixer#857]
Safari supports these two styles of setting line decorations:

```css
.longhand-webkit {
  -webkit-text-decoration-line: overline;
  -webkit-text-decoration-style: double;
  -webkit-text-decoration-color: red;
}

.shorthand-webkit {
  -webkit-text-decoration: overline double red;
}
```

**Live demo:** https://jsbin.com/misoyuq/edit?css,output (I tested in Safari for iOS)

And yet, when writing the standard versions:

```css
.longhand {
  text-decoration-line: overline;
  text-decoration-style: double;
  text-decoration-color: red;
}

.shorthand {
  text-decoration: overline double red;
}
```

Autoprefixer will generate `-webkit-` prefixes only for the longhand properties, but not for the shorthand. (I’ve tested in http://autoprefixer.github.io/). Is there a reason why the shorthand is ignored?

**Comment by ai** [comment]
Yeap, it is a bug. I will fix and release it tomorrow or in Friday.

**Comment by ai** [comment]
Done 005cc51

**Comment by ai** [comment]
Released in 7.1.2.
